payments/schema/queries.py

```diff
@@ -61,3 +65,27 @@ def resolve_additional_products(self, info, **kwargs):
                 )
 
         return AdditionalProduct.objects.all()
+
+    def resolve_additional_product_services(self, info, **kwargs):
+        services = []
+        service_list = []
+        product_type = kwargs.get("product_type")
+        if product_type:
+            product_type = AdditionalProductType(product_type)
```

Comment:
not sure this line is needed 🤔

Reply:
The value that the resolver is receiving is a `str`, so we need to either (1) cast it to the Enum value
```python
or (2) check against the enum `.name`, `if product_type == AdditionalProductType.FIXED_SERVICE.name`

Reply:
ah, okay, then let's cast it 👍

Reply:
I see, graphene converts its Enums to django-enumfields' Enums when we use them as fields on DjangoObjectTypes, but does not do so when they are just arguments on queries 😢 well, let's just cast, yeah